### PR TITLE
Create end2end.yml

### PR DESCRIPTION
**Pull request by nicHoch**
setup action on master push to rebuild end to end test data

### CODE REVIEW

File: .github/workflows/end2end.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  trigger_testdata_rebuild:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Runs a set of commands using the runners shell
+      - name: update end2end test data
+        run: |
+          wget -qO- http://pub099.cs.technik.fhnw.ch/data/end2end/index.py
```
Comment (samaloney) [inline .github/workflows/end2end.yml:26]: So where is this python code index.py?

File: .github/workflows/end2end.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+          echo "git action done"
+        if: github.repository == 'i4Ds/STIXCore'
```
Comment (samaloney) [inline .github/workflows/end2end.yml:28]: what about forks I guess we don't want it to run?

**Review by samaloney — APPROVED** [review]
This looks good, just wondering how the update of the file on the server is done?

**Comment by nicHoch** [comment]
delete this PR as the git action can also be "installed" within the end2end branch: https://github.com/i4Ds/STIXCore/pull/213

**Pull request closed without merging**